src/pages/assistant/index.tsx

```diff
@@ -80,7 +82,11 @@ export const getServerSideProps = withGlobalData(
               initialToFeature: to,
               initialTransportModesFilter: transportModeFilter,
               trip,
-              departureMode: tripQuery.departureMode,
+              departureMode: tripQuery.departureMode ?? DepartureMode.DepartBy,
+              initialDepartureMode: departureModeToDepartureDate(
```

Comment:
Same comment as above, but here I think it becomes more clear that `departureMode` and `initialDepartureMode` can be confusing, and that `initialDepartureDate` might be easier to follow